tests/autoyast/installation.pm

```diff
@@ -22,6 +22,7 @@ use testapi;
 use utils;
 use power_action_utils 'prepare_system_shutdown';
 use version_utils qw(is_sle is_caasp is_released);
+use opensusebasetest;
```

Comment:
You can actually skip that import and use `$self` as all test modules are inherited from opensusebasetest.

Reply:
Updated code, please have a look.